internal/process_test.go

```diff
@@ -1242,6 +1268,8 @@ func TestExportResultsToFile(t *testing.T) {
 }
 
 func TestFetchProjects(t *testing.T) {
```

Comment:
Could you please add tests for the scenario of many projects, over many pages, that we filter using teamName and projectIds?

Reply:
Ok

Reply:
About teamName and projectIds filter there are no sense to make unit tests here because filter is mocked here